.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+name: publish
+
+on:
+  workflow_call:
+    inputs: 
+      source:
+        required: false
+        type: string
+    secrets:
+      apikey:
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: haya14busa/action-cond@v1
+        id: sourceeval
+        with:
+          cond: ${{ inputs.source == '' }}
+          # default nuget source. TODO change to nuget.org
+          if_true: "https://nuget.cloudsmith.io/lombiq/open-source-orchard-core-extensions/v3/index.json"
+          if_false: "${{ inputs.source == '' }}"
```

Comment:
We talked about this but looking more into it I'm confused why can't we use a similar logic with `${{ secrets.DEFAULT_NUGET_PUBLISH_API_KEY }}` here, not to have to specify the secret?

Reply:
When I tried, a called workflow wouldn't read `secrets.ANYTHING` from the source org, it had to be passed in as an arg.
I maybe wrong there, but that was what I found when I tried

Reply:
Yep, this won't work: https://github.community/t/resuable-called-workflow-environment-variables-secrets-and-trigger-event-access/207723/2